plugins/blip/basics/chartbasicsfactory.js

```diff
@@ -43,14 +44,16 @@ var BasicsChart = React.createClass({
     bgUnits: React.PropTypes.string.isRequired,
     onSelectDay: React.PropTypes.func.isRequired,
     patientData: React.PropTypes.object.isRequired,
+    patientSettings: React.PropTypes.object.isRequired,
     timePrefs: React.PropTypes.object.isRequired,
     updateBasicsData: React.PropTypes.func.isRequired,
+    updateBasicsSettings: React.PropTypes.func.isRequired,
     trackMetric: React.PropTypes.func.isRequired,
   },
   _adjustSectionsBasedOnAvailableData: function(basicsData) {
-    if (_.isEmpty(basicsData.data.reservoirChange.data)) {
+    if (_.isEmpty(basicsData.data.reservoirChange.data) || _.isEmpty(basicsData.data.cannulaPrime.data) || _.isEmpty(basicsData.data.tubingPrime.data)) {
       var siteChangeSection = _.find(basicsData.sections, function(section) {
```

Comment:
Why don't we reference the section directly, à la `basicsData.sections.siteChanges`?